### PR TITLE
Update about.md

### PR DESCRIPTION
**Pull request by bianchini88**
Minor grammar/style fixes

### CODE REVIEW

**Review by bedroesb — APPROVED** [review]
These look like obvious changes, so I think we can go right ahead and merge this!

File: pages/about/about.md [hunk 1]
```diff
@@ -16,21 +16,21 @@ The contents are generated and maintained by the ELIXIR community. More informat
 ## Why are the FAIR principles needed?
 At the heart of FAIR Science lies good data management practice. This is increasingly important as life science research becomes data-intensive and traditional ‘wet labs’ make space for ‘dry (computer) labs’.
 
-With that in mind, the [FAIR Principles](https://www.nature.com/articles/sdata201618) were designed to improve the infrastructure supporting the reuse of research data. The intention of these principles is to improve the **F**indability, **A**ccessibility, **I**nteroperability and **R**euse of digital assets.
+With that in mind, the [FAIR Principles](https://www.nature.com/articles/sdata201618) were designed to improve the infrastructure supporting the reuse of research data. These principles intend to improve the **F**indability, **A**ccessibility, **I**nteroperability and **R**euse of digital assets.
```
Comment (martin-nc) [inline pages/about/about.md:19]: I wonder if people know what is meant by "infrastructure". Also, this second paragraph talks about 
digital assets" but the first paragraph only hints at "data". It's hard to se the connection between the paragraphs. Maybe just put: "With that in mind, the [FAIR Principles](https://www.nature.com/articles/sdata201618) were designed to improve  the **F**indability, **A**ccessibility, **I**nteroperability and **R**euse of data."

**Review by martin-nc — COMMENTED** [review]
Just had one comment, but apart from that it looks okay!

File: pages/about/about.md (hunk 1 above)
Comment (bianchini88) [inline pages/about/about.md:19]:
```suggestion
With that in mind, the [FAIR Principles](https://www.nature.com/articles/sdata201618) were designed to improve the **F**indability, **A**ccessibility, **I**nteroperability and **R**eusability of data.
```

Comment (bianchini88) [reply pages/about/about.md]: Good point. I also changed *Reuse* to *Reusability* for better consistency.

**Review by martin-nc — APPROVED** [review]
Yes, that looks good to me!

**Pull request merged**